build/storage/tests/it/ptf_tests/system_tools/test_platform.py

```diff
@@ -267,8 +267,10 @@ def create_virtio_blk_devices_sequentially(
         )
 
     def delete_virtio_blk_devices(self, devices_handles):
-        for device_handle in devices_handles:
-            device_handle.delete(self.cmd_sender)
+        responses = [
+            device_handle.delete(self.cmd_sender) for device_handle in devices_handles
```

Comment:
Is it possible to return delete result as boolean not string value?

Reply:
paramiko return string but i changed it in delete function